grpcapi/src/main/proto/com/example/grpcapi/service/order_service.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package com.example.grpcapi.service;
+
+import "com/example/commonmodels/order/order.proto";
+import "com/example/grpcapi/reqres/order/create.proto";
+import "com/example/grpcapi/reqres/order/find.proto";
+import "com/example/grpcapi/reqres/order/update_status.proto";
+
+option java_multiple_files = true;
+
+service OrderService {
+    rpc CreateOrder (grpcapi.reqres.order.CreateOrderRequest) returns (grpcapi.reqres.order.CreateOrderResponse);
```

Comment:
```suggestion
    rpc createOrder (grpcapi.reqres.order.CreateOrderRequest) returns (grpcapi.reqres.order.CreateOrderResponse);
```

Reply:
According to naming [conventions](https://cloud.google.com/apis/design/naming_convention#method_names), it should be in PascalCase